GP3.Client/MauiProgram.cs

```diff
@@ -1,10 +1,11 @@
-﻿using DevExpress.Maui;
+﻿using Android.OS;
```

Comment:
Is this include necessary? Will most likely break builds for other platforms

Reply:
Thank you, VS. 🙂